idd/Energy+.idd.in

```diff
@@ -17807,6 +17819,175 @@ SurfaceProperty:SolarIncidentInside,
        \required-field
        \type object-list
        \object-list ScheduleNames
+	   
+SurfaceProperty:LocalEnvironment,
```

Comment:
Fixed. At lease one of them is requied, so the min-fields is 3, but not particularly which.